src/python/CRABInterface/DataWorkflow.py

```diff
@@ -169,6 +169,8 @@ def submit(self, workflow, activity, jobtype, jobsw, jobarch, use_parent, second
         try:
             requestname = '%s:%s_%s' % (timestamp, userhn, workflow)
             schedd_name = self.chooseScheduler(scheddname, backend_urls).split(":")[0]
+            if len(requestname) > 232:
```

Comment:
This will work, but here I don't like hardcoded 232. There is no comment why and reviewing or for others working on same code comments help a lot... So this is configuration parameter, which is also used in regexp, so it should be in 1 place. I liked this article about code review: http://software-carpentry.org/blog/2015/10/code-review-habit.html